src/aind_hcr_data_transformation/zeiss_job.py

```diff
@@ -103,7 +103,10 @@ def _get_voxel_resolution_schema_2(
 
         # Grabbing a tile with metadata from acquisition - we assume all
         # dataset was acquired with the same resolution
-        image_to_acquisition_transform = acquisition_config["images"][0][
+        image_to_acquisition_transform = acquisition_config["data_streams"][
+            0][
+            "configurations"][0][
+            "images"][0][
             "image_to_acquisition_transform"
         ]
```

Comment:
Can we change these lines for this?

`try:
    data_stream = acquisition_config.get("data_streams", [])[0]
    configuration = data_stream.get("configurations", [])[0]
    image = configuration.get("images", [])[0]
    image_to_acquisition_transform = image["image_to_acquisition_transform"]
except (IndexError, AttributeError, KeyError) as e:
    raise ValueError("acquisition_config structure is invalid or missing required fields") from e`
    
   Mostly to improve clarity